requirements.txt

```diff
@@ -1,5 +1,6 @@
 django>=2.2,<2.3
-opencivicdata>=3.1.0
+# Better handling for loading divisions from specified CSV
+https://github.com/opencivicdata/python-opencivicdata/archive/master.zip
```

Comment:
do we want to peg this to a specific version, or have it float like this?

Reply:
Great question. I opened https://github.com/datamade/scrapers-us-municipal/issues/64 and once I finish up https://github.com/datamade/how-to/issues/210, let us pin!